invenio_app_ils/acquisition/mappings/v7/orders/order-v1.0.0.json

```diff
@@ -0,0 +1,59 @@
+{
+  "mappings": {
+    "date_detection": false,
+    "numeric_detection": false,
+    "properties": {
+      "$schema": {
+        "type": "keyword"
+      },
+      "pid": {
+        "type": "keyword"
+      },
+      "document_pid": {
```

Comment:
just to clarify: shouldn't the order have "ForeignKey" to `document_request`, not only `document`? Then we wouldn't have to include patron_pid

Reply:
I think that we still have to solve the integration with the request workflow... we can address this in one of the future PRs.